### PR TITLE
API for love leaderboard.

### PR DESCRIPTION
**Pull request by sharan1**
Hi,
As per my discussion with one of the members of Yelp Love (cced: love@yelp.com), I have written an API for getting love leaderboard data. Kindly let me know, if this require any changes.

Thanks,

### CODE REVIEW

File: logic/leaderboard.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+from datetime import timedelta
+
+from logic import TIMESPAN_LAST_WEEK
+from logic import TIMESPAN_THIS_WEEK
```
Comment (rockdog) [inline logic/leaderboard.py]: TIMESPAN_THIS_WEEK is not used.

File: views/api.py [hunk 2]
```diff
@@ -3,9 +3,12 @@
 from flask import request
 
 from errors import TaintedLove
+from logic import TIMESPAN_LAST_WEEK
```
Comment (rockdog) [inline views/api.py]: TIMESPAN_LAST_WEEK is not used.

Comment (sharan1) [reply views/api.py]: Removed

File: logic/leaderboard.py (hunk 1 above)
Comment (sharan1) [reply logic/leaderboard.py]: Removed

**Pull request merged**